readme.md

```diff
@@ -54,18 +54,18 @@ const fs = require('fs');
 Or from a remote location:
 
 ```js
-const FileType = require('file-type');
-const http = require('http');
+const https = require('https');
+const fileType = require('file-type');
```

Comment:
```suggestion
const FileType = require('file-type');
```

Reply:
Note that I have updated the examples utilizing [node-fetch](https://www.npmjs.com/package/node-fetch).

It keeps the example clean, minimizes the difference between Node.js and browser.